.jshintrc

```diff
@@ -14,12 +14,13 @@
   "quotmark": "single",
   "regexp": true,
   "undef": true,
-  "unused": true,
+  "unused": "vars",
```

Comment:
If you have a signature that requires (x,y) but you only use y jshint will fail. I also prefer having the entire function signature since it makes code more readable.